subcommand/common/common.go

```diff
@@ -30,3 +32,16 @@ func Logger(level string) (hclog.Logger, error) {
 		Output: os.Stderr,
 	}), nil
 }
+
+// ValidatePort converts flags representing ports into integer and validates
+// that it's in the port range.
+func ValidatePort(flagName, flagValue string) error {
+	port, err := strconv.Atoi(flagValue)
+	if err != nil {
+		return errors.New(fmt.Sprintf("%s value of %s is not a valid integer.", flagName, flagValue))
+	}
+	if port <= 1023 || port > 65353 {
```

Comment:
Bonus points if you can find a library that does this validation, I looked and couldn't find one.
In lieu of reading `/proc/sys/net/ipv4/ip_local_port_range`, I think this is safe for now.
In theory this may be an invalid range based on how the kernel/network is configured.

Reply:
Also a really dumb question on my part but did you mean `65535` instead of `65353`?

Reply:
Oh you're absolutely right I don't know where my number came from, changed that! Yeah it would be nice to use a library but yeah I also couldn't find one.